src/borg/archive.py

```diff
@@ -1877,8 +1877,11 @@ def valid_item(obj):
                 return
         num_archives = len(archive_infos)
 
+        pi = ProgressIndicatorPercent(total=num_archives, msg='Checking archives %3.1f%%', step=0.1,
+                                      msgid='archive.robust_iterator')
```

Comment:
Maybe easier to make sense of:
```
msgid='check.rebuild_refcounts'
```

Reply:
Other than that, guess we are fine.

Reply:
A PR for 1.1-maint would be welcome, thanks!

Reply:
> A PR for 1.1-maint would be welcome, thanks!

Done: #5810